@navikt/core/tokens/darkside/tests/validate-config.test.ts

```diff
@@ -0,0 +1,87 @@
+import { describe, expect, test } from "vitest";
+import { globalScale } from "../create-configuration";
+import { contrastTokenConfig } from "../tokens/contrast";
+import { neutralTokenConfig } from "../tokens/neutral";
+import { radiusTokenConfig } from "../tokens/radius";
+import { semanticTokenConfig } from "../tokens/semantic";
+import { semanticTokensForAllRolesConfig } from "../tokens/semantic-roles";
+import { spacingTokenConfig } from "../tokens/spacing";
+
+const ConfigKeysWithGroup = ["value", "type", "group"];
+const ConfigKeys = ["value", "type"];
```

Comment:
Should these start with lower case letter?
```suggestion
const configKeysWithGroup = ["value", "type", "group"];
const configKeys = ["value", "type"];
```